userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -0,0 +1,161 @@
+
+==== 1. Request a Certificate
+
+If you are setting up a new multi-tier or CloudFormation implementation, choose the HTTPS option for your load balancer. You will come to a page that asks for your certificate. click on *Request a new Certificate from ACM*. 
+
+Otherwise, to add HTTPS security to an existing implementation, select *Security, Identity, and Compliance*, then *Certificate Manager* from the Services menu. In this case you will need to edit your load balancer(s) later on.
+
+In either case you will be taken to the main page of the ACM. Initially this will probably show that you have no certificates. On the left menu, click *Request a Certificate*.
+
+The next page will give you the option to request a public or private certificate. The private option may be grayed out. Click *Request a Public Certificate*. The page that appears will ask you to specify several parameters, including one or more domain names and a validation method.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-request-certificate.png[align=center]
+
+==== 2. Setup a CNAME
+
+To validate your certificate, you must first setup a *validation* CNAME. The values to use for the CNAME will be seen shortly.  You will then create the CNAME that you will actually use to access the system.
+
+Enter your domain name, using the wildcard format (e.g., `*.mydomain.com`. In the second pane, select *DNS Validation*. Do not change the other settings. Scroll to the bottom of the page, and click *Request*. The main ACM page will again appear.
+
+You should see a new certificate in progress with your (wildcard) domain name. It's status will be *Pending Validation*. Select the Certificate ID. The display will expand to show the values that you must use to create your validation CNAME.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-cert-validation.png[align=center]
+
+The two strings labeled `Name` and `Value` should be copied to the two required entries for your CNAME. Copy these long strings carefully. Do not include any trailing periods.
+
+Your second CNAME is much simpler. The `Name` entry will be `kaui`. The `Value` entry will be simply your domain name, e.g., `mydomain.com`.
+
+==== 3. Create and Validate your Certificate
+
+When your CNAMEs are set, return to ACM and click *Continue*. The display changes to show that your certificate is *Pending Validation*. The certificate should now appear also on the main ACM page. When (and if) it is validated, the status will change to *Success*. *This may take several hours or more.* A short time later it will change again to *Issued*.
+
```

Comment:
Again, wondering if a screenshot might help?

Reply:
Perhaps. These screens have already been seen, though, except the status value is different.

Reply:
okay, it's difficult to visualize what the final pages would look like while reviewing here in Github. I would suggest merging this PR and doing another pass.